File: SDKs/amazonaws.com/securityhub/2018-10-26/java/lib/src/main/java/openapisdk/models/shared/AwsRedshiftClusterElasticIpStatus.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * AwsRedshiftClusterElasticIpStatus
 * The status of the elastic IP (EIP) address for an Amazon Redshift cluster.
**/
public class AwsRedshiftClusterElasticIpStatus {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("ElasticIp")
    public String elasticIp;
    public AwsRedshiftClusterElasticIpStatus withElasticIp(String elasticIp) {
        this.elasticIp = elasticIp;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("Status")
    public String status;
    public AwsRedshiftClusterElasticIpStatus withStatus(String status) {
        this.status = status;
        return this;
    }
}

File: SDKs/amazonaws.com/securityhub/2018-10-26/python/src/sdk/models/shared/awsredshiftclusterelasticipstatus.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class AwsRedshiftClusterElasticIPStatus:
    r"""AwsRedshiftClusterElasticIPStatus
    The status of the elastic IP (EIP) address for an Amazon Redshift cluster.
    """
    
    elastic_ip: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('ElasticIp') }})
    status: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Status') }})
    
